### ISSUE TITLE
WARNING scrobbler::invalid_service[last.fm]

### ISSUE THREAD

**Issue by alecvn** [mariusor/mpris-scrobbler#60]
I'm getting `WARNING scrobbler::invalid_service[last.fm]` when scrobbling from Ubuntu.  I've reinstalled the mpris-scrobbler a few times and followed the instructions.  This doesn't happen for me from work where I have a much older version of mpris-scrobbler.

`$ mpris-scrobbler -vvv ` yields:
https://pastebin.com/23CnJsyY

**Comment by mariusor** [comment]
I think this signifies a problem with your last.fm credentials. Can you paste the content of your `~/.local/share/mpris-scrobbler/credentials` (making sure you obfuscate token, session - and maybe username) ?

I assume that if you try to authenticate to last.fm again it will work.

I will make a note to improve the error message if that's the case, though. :)

**Comment by alecvn** [comment]
Ah yes, that file proved you correct.

```
[lastfm]
enabled = false
```

And after authenticating again:

```
[lastfm]
enabled = true
token = {TOKEN}
```

I guess I should open another issue for this, but the reason I re-installed mpris-scrobbler and was checking the logs is because I it hasn't been working for me for a few weeks now.  As stated, my version of mpris-scrobbler at work is still scrobbling (apart from the first track I select, as noted in issue #53), but every version I've tried since then at home hasn't been working.

**Comment by mariusor** [comment]
OK, I'm glad it worked. I guess better error messages are needed for this use case.

For a second I was afraid that last.fm changed something in their authentication mechanism and I would need to update.  :)

**Comment by alecvn** [comment]
I think maybe just adding something to the README so users know to check in `~/.local/share/mpris-scrobbler/credentials` if things don't work as they except should be fine for now.  For instance, after I enabled and got my token for last.fm, I ran the command to start a session.  That failed silently.  I had no idea it failed until I looked at your message here again where you mention that `session` should be in there.  So I ran the session start command again and double checked the credentials file to make sure it succeeded.  Then everything was fine!
